src/Steam.package/STExampleApp.class/instance/applicationClasses.st

```diff
@@ -0,0 +1,4 @@
+initialization
+applicationClasses
+
+	^ { STExampleArticle. STGameCategory }
```

Comment:
Maybe in the superclass we should have this same method defined as `#subclassResponsibility`. Like that people defining their own app know what are the methods they have to define. no?

Reply:
already done